martinoywa/numbers-tutor
# Cu apybj0 front end tests joe mindo

### What does this PR do?
- **Creates the frontend tests**

### How can this be manually tested?
- Install [Cypress](https://docs.cypress.io/guides/getting-started/installing-cypress.html#System-requirements)
- Start your server
- Type the following command ```node_modules/.bin/cypress open```
- You should see a screen such as 
![Screenshot from 2020-12-10 02-59-21](https://user-images.githubusercontent.com/47358442/101703820-66d8ff00-3a94-11eb-8e50-0c0c68acacf1.png)
- Click on the files named, ```home_page_spec.js write_page_spec.js and speak_page_spec.js```
- For each file,
![Screenshot from 2020-12-10 03-02-44](https://user-images.githubusercontent.com/47358442/101703920-a869aa00-3a94-11eb-951c-21a76ad25a0e.png)
 a window showing the results should pop up
